www/securestorage.js

```diff
@@ -108,6 +129,10 @@ SecureStorage.prototype = {
 };
 
 if (cordova.platformId === 'android') {
+    SecureStorage.prototype.options = {
+        packageName: null
+    }
+
```

Comment:
Please change the above to read as follows and move it to the SecureStorage prototype definition
```
options: {
   android: {
        packageName: null
    }
}
```

Reply:
Done. This required additional changes during the `_merge_options`.